Automattic/themes
# Modern Business: Remove Full Site Editing Support

Follow up to #1351

#### Changes proposed in this Pull Request:

Remove Full Site Editing support from Modern Business in order to fully replace it with Maywood (and later with the other Varia-based themes too).

#### Testing instructions:

See https://github.com/Automattic/wp-calypso/pull/35856

After applying #35856, activate Modern Business and smoke test it.
- The front end should look roughly like the [theme demo](https://modernbusinessdemo.wordpress.com/).
- The page editor shouldn't have any weird things happening (except that, currently, it displays the template parts placeholders, but it won't load them).

#### Related issue(s):

See https://github.com/Automattic/wp-calypso/issues/35939
